config/config_test.go

```diff
@@ -0,0 +1,165 @@
+package config
+
+import (
+	"os"
+	"testing"
+)
+
+func mustTempFile(t *testing.T) string {
+	f, err := os.CreateTemp(t.TempDir(), "pem-*.crt")
+	if err != nil {
+		t.Fatalf("temp file: %v", err)
+	}
+	f.Close()
+	return f.Name()
+}
+
+// ---------------------------- Positive cases ----------------------------
+func TestLoadConfigPositiveVariants(t *testing.T) {
+	ca := mustTempFile(t)
+	cert := mustTempFile(t)
+	key := mustTempFile(t)
```

Comment:
These files don't ever seem to be cleaned up.

Reply:
added `t.Cleanup` to remove tmp files